extensions/amp-bind/0.1/amp-state.js

```diff
@@ -48,8 +49,11 @@ export class AmpState extends AMP.BaseElement {
 
   /** @override */
   buildCallback() {
-    user().assert(isExperimentOn(this.win, 'amp-bind'),
-        `Experiment "amp-bind" is disabled.`);
+    // Don't check experiment when testing.
```

Comment:
I don't think I've seen it anywhere else. I don't think a short explanation here is too much to ask.